bin/bundle

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env ruby
```

Comment:
Are the bin files supposed to be checked in? I genuinely do not know best practices regarding that.

Reply:
according to stackoverflow and heroku, bin/ should not be ignored and should be included in the repo.